modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/EditableProcessorContext.js

```diff
@@ -15,12 +15,12 @@
 import React, {useCallback, useContext, useReducer} from 'react';
 
 const EditableProcessorContext = React.createContext(null);
-const INITIAL_STATE = null;
+const INITIAL_STATE = {editableClickPosition: null, editableUniqueId: null};
 const SET_EDITABLE_UNIQUE_ID = 'SET_EDITABLE_UNIQUE_ID';
 
 function reducer(state = INITIAL_STATE, action) {
 	if (action.type === SET_EDITABLE_UNIQUE_ID) {
-		return action.editableUniqueId;
+		return action;
```

Comment:
shouldn't it be returning something like 
```javascript
{
   editableUniqueId: action.editableUniqueId, 
   editableClickPosition: action.editableClickPostion
}
```
otherwise we are including the type of the action in the state which we are not be using, right?